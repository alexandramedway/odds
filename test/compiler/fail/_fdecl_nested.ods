do sum = (x, y, z) ->
    do aux = (a, b) -> return a + b
    return aux(x, y) + z

do sum(1, true, 3)
