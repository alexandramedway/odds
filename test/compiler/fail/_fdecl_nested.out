[31mSemantic error[0m, line 1:
  Function 'sum' expected argument of type Num but was passed Bool instead
